### DISCUSSION TITLE
Lesson 13 Full Stack Defi <ValueError: constructor Sequence has incorrect length, expected 1 but got 0>

### DISCUSSION THREAD

**Discussion by bubu59** [smartcontractkit/full-blockchain-solidity-course-py#1337] [Q&A]
Hi Guys, I'm new to this! Appreciate any advice on the current issue I'm facing at the moment. 

<Context/Issue>
At timestamp: 13:47:50 of the course, when I ran <brownie run/scripts deploy.py> , I run into an error that says:
<ValueError: constructor Sequence has incorrect length, expected 1 but got 0>

![image](https://user-images.githubusercontent.com/74093051/160223983-f9bcb147-0310-4363-a952-9ee01a6a06e3.png)

**Comment by cromewar** [comment]
Hello @bubu59 please refer to #980 in order to know how to make question.
Please share your code In order we can review it.

**Reply by bubu59** [reply]
Roger that!
